### DISCUSSION TITLE
DevTools failed to load source map: Could not parse content

### DISCUSSION THREAD

**Discussion by Grafdude** [parcel-bundler/parcel#6543] [Q&A] [answered]
I've been having this issue with Parcel lately with the source maps. I get this error:

Unexpected token < in JSON at position 0

I have absolutely no idea why it is doing that, parcel is working, however without the mapping functioning, it is very hard to work without it.  Looking up the css styles in the devTools not knowing from which sass file the style is coming from is really a pain in the rear. 

edit: The source maps are working just fine with the previous beta version v2.0.0. beta-1 and with using the latest sass. This issue could be related to the "parcel sass transformer" that has been introduced?

**Comment by DeMoorJasper** [comment, marked as answer]
This has been fixed in the latest nightly,  it wasn't being served by the dev server before

**Reply by Grafdude** [reply]
Thank you very much!
